frontend/src/Buildings/actionCreator.js

```diff
@@ -3,6 +3,9 @@ import { SERVER_URL } from 'react-native-dotenv';
 export const FETCH_BUILDINGS_REQUEST = 'fetchBuildingsRequest';
 export const FETCH_BUILDINGS_SUCCESS = 'fetchBuildingsSuccess';
 export const FETCH_BUILDINGS_FAILURE = 'fetchBuildingsFailure';
+export const ADD_FARM = 'addFarm';
```

Comment:
```suggestion
export const ADD_BUILDING_REQUEST = 'addBuildingRequest';
export const ADD_BUILDING_SUCCESS = 'addBuildingSuccess';
export const ADD_BUILDING_FAILURE = 'addBuildingFailure';
```

why aren't you handling these cases? request should change the `isLoading` state to true, etc..... just like with every other action